sbt-ash/src/test/scala/org/virtuslab/ash/DumpPersistenceSchemaSpec.scala

```diff
@@ -4,12 +4,12 @@ import better.files.File
 import org.scalatest.flatspec.AnyFlatSpecLike
 import org.scalatest.matchers.should
 
-class DumpEventSchemaSpec extends AnyFlatSpecLike with should.Matchers {
+class DumpPersistenceSchemaSpec extends AnyFlatSpecLike with should.Matchers {
```

Comment:
Some final leftovers, not including README:

```
$ ggr -i 'dump.*event'
dump-persistence-schema-compiler-plugin/src/test/scala/org/virtuslab/ash/DumpPersistenceSchemaCompilerPluginComponentSpec.scala:31:  "DumpPersistenceSchemaPlugin with correct Event[_,_]" should {
sbt-ash/src/test/scala/org/virtuslab/ash/DumpPersistenceSchemaSpec.scala:8:  "DumpEventSchemaPlugin" should "dump event schema based on provided jsons" in {
```

Reply:
Oh, the first one isn't valid either, so just this line below in `DumpPersistenceSchemaSpec`